src/index.js

```diff
@@ -12,11 +12,14 @@ export const renderReact = (name, component) => hypernova({
   },
 
   client() {
-    const { node, data } = load(name);
+    const payloads = load(name);
 
-    if (node) {
-      const element = React.createElement(component, data);
-      ReactDOM.render(element, node);
+    if (payloads) {
+      Array.prototype.map.call(payloads, (payload) => {
```

Comment:
I doubt `hypernova` change the `load` API in 2.0.0 ?
https://github.com/airbnb/hypernova/blob/master/src/index.js#L81-L88